kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -4,11 +4,18 @@
     <div v-if="currentChannelIsCustom">
       <CustomContentRenderer :topic="topic" />
     </div>
-
-    <div v-else class="page">
+    <!-- appearanceOverrides overrides the default page styling -->
```

Comment:
Presumably this is going to need a rebase on top of @nucleogenesis's topics refactor?

Reply:
yes